core/templates/cards.html

```diff
@@ -103,39 +119,40 @@ <h4 class="modal-title">Col-League</h4>
         timer.start();
     }
 
+    function updateStars(starIndex){
+        $('#0star-empty').removeClass('glyphicon-star-empty');
+        $('#0star-empty').addClass('glyphicon-star');
+    }
+
     function nextRound() {
         var cards = '{{ cards }}';
-
-        $.get('/rounds/', {matrix: cards, round: round, score: score}, function (data) {
+        formlessAjaxPostCSRFSetup();
+        $.post('/rounds/', {matrix: cards, round: round, score: score, cardIndex: selectedCardIndex},
+        function (data) {
```

Comment:
when using post requests outside of a form, you have to send the the csrf token along with the request or the django server will assume its an unsafe request